### ISSUE TITLE
HTML5 Builds failing to preload

### ISSUE THREAD

**Issue by SlickFromMars** [openfl/lime#1632]
<img width="960" alt="image" src="https://user-images.githubusercontent.com/82390475/218114951-31053ea7-cc5d-4518-8165-a91cc13e645f.png">

When compiling my HaxeFlixel project to html5 using lime, the preloader gets stuck here when I click on the html5 file. I am aware that it will work when I use the `lime run html5` command in the root, but I want to deploy the app to a website and I don't want users to run into this issue.

Does anyone have a solution?

Source code is [here](https://github.com/SlickFromMars/rooms-2022) if you need it.

**Comment by player-03** [comment]
Try pressing ctrl+shift+I (or right-click → inspect) to open the developer tools. The console tab will probably have an error message with more details.

Odds are, the problem is due to security restrictions preventing you from accessing your asset files. `lime run` gets around this by starting an HTTP server, serving all the files at once as a website would do. If you want to test without `lime run`, you can [use Python to start a server manually](https://www.geeksforgeeks.org/setting-up-a-simple-http-server-using-python/).

**Comment by joshtynjala** [comment]
> Odds are, the problem is due to security restrictions preventing you from accessing your asset files.

Yeah, Lime/OpenFL pretty much always needs to be run from a server. You can't open the .html file from your hard drive because web browsers are really strict about accessing local files (images, etc.) and things. Back in the day, there wasn't much difference between local vs server, but at some point, browser vendors really locked things down for local files.

> I want to deploy the app to a website and I don't want users to run into this issue.

It sounds like you'll be deploying to a server, so you shouldn't run into this issue.
